common/djangoapps/util/views.py

```diff
@@ -56,6 +56,39 @@ def jsonable_server_error(request, template_name='500.html'):
         return server_error(request, template_name=template_name)
 
 
+def handle_500(template_path):
```

Comment:
@saleem-latif to make this decorator more flexible/useful I suggest to pass in the context also.